src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -114,13 +92,44 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
   }
 
   /**
+   * Gets the current account.
+   *
+   * @return AccountInterface
+   */
+  public function getAccount() {
```

Comment:
The current user should actually already be injected into the default selection handler. We could use that.